server/routerlicious/packages/routerlicious-base/src/alfred/services/deltaService.ts

```diff
@@ -25,22 +30,39 @@ export class DeltaService implements IDeltaService {
 		from?: number,
 		to?: number,
 	): Promise<ISequencedDocumentMessage[]> {
-		// Create an optional filter to restrict the delta range
-		const query: any = { documentId, tenantId };
-		if (from !== undefined || to !== undefined) {
-			query["operation.sequenceNumber"] = {};
+		const metricProperties = {
+			[BaseTelemetryProperties.tenantId]: tenantId,
+			[BaseTelemetryProperties.documentId]: documentId,
+			from,
+			to,
+		};
+		const metric = Lumberjack.newLumberMetric(
+			LumberEventName.GetDeltasFromDb,
+			metricProperties,
+		);
+		try {
+			// Create an optional filter to restrict the delta range
+			const query: any = { documentId, tenantId };
+			if (from !== undefined || to !== undefined) {
+				query["operation.sequenceNumber"] = {};
 
-			if (from !== undefined) {
-				query["operation.sequenceNumber"].$gt = from;
-			}
+				if (from !== undefined) {
+					query["operation.sequenceNumber"].$gt = from;
+				}
 
-			if (to !== undefined) {
-				query["operation.sequenceNumber"].$lt = to;
+				if (to !== undefined) {
+					query["operation.sequenceNumber"].$lt = to;
+				}
 			}
-		}
 
-		const sort = { "operation.sequenceNumber": 1 };
-		return this.queryDeltas(collectionName, query, sort);
+			const sort = { "operation.sequenceNumber": 1 };
+			const deltas = this.queryDeltas(collectionName, query, sort);
+			metric.success("getDeltasFromDb succeeded");
+			return deltas;
+		} catch (err) {
+			metric.error("getDeltasFromDb failed");
```

Comment:
It might be helpful to include the error with the metric
`metric.error("getDeltasFromDb failed", err)`